apps/swirl-docs/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "swirl-docs",
+  "version": "0.1.0",
+  "private": true,
```

Comment:
Please add `description`, `"author": "Flip GmbH"`, `"repository": "https://github.com/getflip/swirl"`, `  "license": "SEE LICENSE IN LICENSE.md"`, `bugs` and remove `private`